tools/dockerversioning/README.md

```diff
@@ -0,0 +1,70 @@
+# Description
+
+Versioning tools for Dockerfile source repos.
+
+- `dockerfiles` generates versionsed Dockerfiles from a common template.
+- `cloudbuild` generates a configuration file to build these Dockerfiles using
+  [Google Container Builder](https://cloud.google.com/container-builder/docs/).
+
+# Installation
+
+- Install [Bazel, version 0.13.0](https://bazel.build) as the build tool.
```

Comment:
As far as I see in dockerfile we're using Bazel 0.19 , no?

Reply:
Right.